Worksheet-B/README.MD

```diff
@@ -0,0 +1,26 @@
+Pseudocode:
+```
+Generate 10 words which are 5 letters long
+Select one word at random as the answer, this word = chosenword
+Display the 10 words
+
+sequence starts here
+
+ If  
+  userinput = chosenword
+  unlock terminal
+  end if
+  
+ elseif 
+  the user has <3 strikes 
+  tell user number of mutual letters of userinput and chosenword
```

Comment:
The worksheet asked you to provide pseudocode to calculate this, e.g. by looping over the words and counting the mutual letters one by one.